guide/.vuepress/components/DocsLink.vue

```diff
@@ -9,8 +9,8 @@ import { computed } from 'vue';
 import { defaultDjsBranch } from '../constants';
 
 const legacyBaseURL = 'https://discord.js.org/#/docs';
-const baseURL = 'https://discordjs.dev/docs/packages';
-const sections = ['discord.js', 'brokers', 'builders', 'collection', 'core', 'formatters', 'proxy', 'rest', 'util', 'voice', 'ws'];
+const baseURL = 'https://discord.js.org/docs/packages';
+const sections = ['discord.js', 'brokers', 'builders', 'collection', 'core', 'formatters', 'next', 'proxy', 'rest', 'util', 'voice', 'ws'];
```

Comment:
Shouldn't add the /next package—that's not for this iteration of the guide (we will very likely never reference anything to a newer version of discord.js!)

Reply:
Ah alright, I thought the next package would have more added later—kinda missed the purpose then.